docs/cli/usage/mock.md

```diff
@@ -100,6 +99,73 @@ query {
 }
 ```
 
+### Function mock environment variables
+When mocking Lambda functions, Amplify populates environment variables that mimic what will be present when deployed in the cloud (see [function mock limitations](#function-mock-limitations)). Amplify will also parse the CloudFormation template of the function and attempt to resolve any environment variables specified there.
+
+Specifically, CloudFormation parameters will be resolved by:
+1. Resolving values specified in the `parameters.json` file for the function
+2. Resolving values specified in `team-provider-info.json` within the `<env>.categories.function.<function name>` block, where `<env>` is the currently checked out environment and `<function name>` is the function being mocked
+3. `AWS::Region`, `AWS::AccountID`, `AWS::StackName`, and `AWS::StackId` are resolved by parsing the `awscloudformation` configuration of the current environment in `team-provider-info.json`
+4. Parameters constructed from depending on other resources in the project are resolved by parsing the `amplify-meta.json`. Additionally, if a mock API is currently running and the function depends on the API, the local API URL will be substituted for the cloud URL
+
+In addition to using the context above to resolve CloudFormation template values, the mock environment will populate [lambda runtime environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime) in the following way:
+1. `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN` will be populated using the AWS credentials that the CLI is currently configured to use
+2. `_HANDLER`, `AWS_REGION`, `AWS_LAMBDA_FUNCTION_NAME`, `LAMBDA_TASK_ROOT`, and `LAMBDA_RUNTIME_DIR` will be set based on the function being mocked
+3. Static defaults will be specified for all other runtime environment variables. The full list of static defaults can be found [here](https://github.com/aws-amplify/amplify-cli/blob/af76446d18bf626ca5f91c3ad41081175c959807/packages/amplify-util-mock/src/utils/lambda/populate-lambda-mock-env-vars.ts#L39)
+
+If any of the default environment values do not work for your mocking use case, you can also specify overrides in a `.env` file within the function directory (ie. `<project root>/amplify/backend/function/<function name>/.env`).
+
+### Connecting to a mock @model table
+If you have a function that connects to a table generated by @model you may want to test out this code against the mock table when running the mock API. To do that, create a `.env` file in the function directory and add the following:
```

Comment:
```suggestion
If you want to test a function to connects to a mocked table generated by @model, create a `.env` file in the function directory and add the following:
```
reword this, not happy with my wording either but current approach seems verbose.

Reply:
Reworded to "Connect a mock function that operates on a table generated by `@model` to the mock table when running `amplify mock api` by:"

Still seems a bit convoluted by it's at least a shorter sentence to grok